locales/base/translation.json

```diff
@@ -1248,6 +1248,8 @@
     "buttonLabel": "Tell me more"
   },
   "dappsDisclaimerAllDapps": "{{appName}} is not affiliated with these dapps and makes no representations or claims in using them. Please acknowledge the risk before using a dapp by reviewing the user agreement or privacy policy.",
+  "dappsDisclaimerSingleDapp": "{{appName}} is not affiliated with this dapp and makes no representations in using it. Always review a dapps user agreement or privacy policy.",
+  "dappsDisclaimerUnlistedDapp": "This dapp is not listed in {{appName}}. {{appName}} is not affiliated with this dapp and makes no representations in using it. Always review a dapps user agreement or privacy policy.",
```

Comment:
Is the "dapps" not supposed to have an apostrophe?

Reply:
nice catch!! i think it should have one indeed..i updated :D